CHANGELOG.md

```diff
@@ -1,4 +1,67 @@
 # osquery Changelog
+<a name="5.5.0"></a>
+## [5.5.0](https://github.com/osquery/osquery/releases/tag/5.5.0)
+
+[Git Commits](https://github.com/osquery/osquery/compare/5.4.0...5.5.0)
+
+Osquery 5.5.0 has some really exciting table updates! There is much
```

Comment:
```suggestion
Osquery 5.5.0 has some really exciting table updates! There is a much
```

Reply:
you think "the" not "a" ?

Reply:
either way! But I would write it's "a much anticipated"